app/views/multifactor_auths/_mfa_setup.html.erb

```diff
@@ -0,0 +1,34 @@
+<% @title = t("multifactor_auths.#{_context}.title") %>
+
+<div class="t-body l-overflow">
+  <div class="l-half--l l-half--l--center">
+    <%= raw @qrcode_svg %>
+  </div>
+  <div>
+    <p><%= t('.scan_prompt') %></p>
+    <% if _context == "replace"%>
+      <p><%= t('multifactor_auths.replace.timeout_warning') %>
+     <% end %>
+    <p>
+      <span id="mfa-account"><%= t('.account', account: "#{issuer}:#{current_user.email}") %></span><br>
+      <span id="mfa-key"><%= t('.key', key: @seed.chars.each_slice(4).map(&:join).join(' ')) %></span><br>
```

Comment:
I implemented this in my most recent commit; I figured it would be bad practice to put it in the erb file, so I had to put it in the controller and pass it through the partial, which seems like a lot of mental overhead for something that probably won't change - what do you think?